### PR TITLE
Replace unified search text field busy indicator with custom indicator

### PR DESCRIPTION
**Pull request by claucambra**
Addresses a point in #4434 

New spinner:

https://user-images.githubusercontent.com/70155116/179546285-9cf6a3bc-f8b1-439c-92a8-ddb6e4a87669.mov

Old spinner:

https://user-images.githubusercontent.com/70155116/179546416-f1d7a79c-22b3-46b1-bcb6-f9f5e591e0a0.mov

### CODE REVIEW

**Review by allexzander — APPROVED** [review]

File: src/gui/tray/UnifiedSearchInputContainer.qml [hunk 1]
```diff
@@ -70,7 +69,23 @@ TextField {
...
+    }
+
+    RotationAnimator {
+        target: busyIndicator
+        running: trayWindowUnifiedSearchTextField.isSearchInProgress
+        from: 0
+        to: 360
+        loops: Animation.Infinite
+        duration: 3000
     }
```
Comment (mgallien) [inline src/gui/tray/UnifiedSearchInputContainer.qml:89]: this should be a child of the `Image` component to directly get an hint that they are linked

**Review by mgallien — COMMENTED** [review]
I would have kept the BusyIndicator component and just style its `contentItem` to have it easier to understand that we just have a plain `BusyIndicator` but with a custom style

**Comment by claucambra** [comment]
> I would have kept the BusyIndicator component and just style its `contentItem` to have it easier to understand that we just have a plain `BusyIndicator` but with a custom style

Sure -- I separated it into its own file and it is now a BusyIndicator

File: src/gui/tray/UnifiedSearchInputContainer.qml (hunk 1 above)
Comment (claucambra) [reply src/gui/tray/UnifiedSearchInputContainer.qml]: Sure, made it the child of the image

File: src/gui/tray/SyncStatus.qml [hunk 2]
```diff
@@ -31,19 +31,10 @@ RowLayout {
...
-        target: syncIcon
-        running:  syncStatus.syncing
-        from: 0
-        to: 360
-        loops: Animation.Infinite
-        duration: 3000
+        padding: 0
+
+        imageSource: syncStatus.syncIcon
+        running: syncStatus.syncing
```
Comment (mgallien) [inline src/gui/tray/SyncStatus.qml:37]: I guess you also want to set its `visible` property
or do we want to always have it visible

Comment (claucambra) [reply src/gui/tray/SyncStatus.qml:37]: We always want it to be visible (otherwise the checkmark icon for 'All synced!' would disappear

**Review by mgallien — APPROVED** [review]

**Pull request merged**